.history/force-app/main/default/classes/NewOfferConsistencyCheckTest_20210713113245.cls

```diff
@@ -0,0 +1,102 @@
+/** 
+ * @author Mikołaj Lechtański <mikolaj.lechtanski@accenture.com> 
+ * @date 07.07.2021 
+ * @description Test class checking NewOfferConsistencyCheck trigger
+**/
+@isTest
+private class NewOfferConsistencyCheckTest {
+	@isTest 
+  static void insertOfferVehiclePresent() {
```

Comment:
jdock?

Reply:
dodałem niepotrzebniej jakąś wtyczkę gitową i powrzucało historyczne pliki z folderu ".history". Naprawię to przez .gitignore